intellst-api/config/routes.yaml

```diff
@@ -1,3 +1,4 @@
-#index:
-#    path: /
-#    controller: App\Controller\DefaultController::index
+
+login_check:
+  path:     /login_check
+  methods:  [POST]
```

Comment:
please add blank line in the end of the file (it is a convention that helps detecting changes in files correctly)

Reply:
done